src/unused.js

```diff
@@ -0,0 +1 @@
+console.log('Ooops!);
```

Comment:
It looks like there is a missing closing quote for the string 'Ooops!'. This will cause a syntax error. Make sure to close the string with a matching quote.